draft-ietf-rats-daa.md

```diff
@@ -79,54 +79,85 @@ A PKIX Certificate is an X.509v3 format certificate as specified by {{RFC5280}}.
+   '----------'                                '---------------'
 ~~~~
-{: #dataflows title="DAA data flows"}
+{: #sign-mapping title="RATS Architecture for the DAA-Signing Protocol"}
+
+The DAA Issuer acts as the Endorser for the Group Public Key that is used by the Verifier for the appraisal of evidence of anonymized Attesters that use the DAA credentials and associated key material to produce Evidence.
 
-DAA {{DAA}} is a signature scheme that allows the privacy of users that are associated with an Attester (e.g. its owner) to be maintained.
+In consequence, DAA provides a signature scheme that allows the privacy of users that are associated with an Attester (e.g., its owner) to be maintained.
 Essentially, DAA can be seen as a group signature scheme with the feature that given a DAA signature no-one can find out who the signer is, i.e., the anonymity is not revocable.
 To be able to sign anonymously, an Attester has to obtain a credential from a DAA Issuer.
-The DAA Issuer uses a private/public key pair to generate credentials for a group of Attesters <!-- this could be phrased a bit confusing as below it is stated that the key-pair is used for a group of Attesters --> and makes the public key (in the form of a public key certificate) available to the verifier in order to enable them to validate the Evidence received.
+The DAA Issuer uses a private/public key pair to generate credentials for a group of Attesters <!-- this could be phrased a bit confusing as below it is stated that the key-pair is used for a group of Attesters --> and makes the public key (in the form of a public key certificate) available to the Verifier in order to enable it to validate the Evidence received.
 
 In order to support these DAA signatures, the DAA Issuer MUST associate a single key pair with a group of Attesters <!-- is it clear enough what exactly "a group of Attesters" means? --> and use the same key pair when creating the credentials for all of the Attesters in this group.
-The DAA Issuer’s group public key certificate replaces the individual Attester Identity documents during authenticity validation as a part of the appraisal of Evidence conducted by a verifier.
+The DAA Issuer’s group public key certificate replaces the individual Attester Identity documents during authenticity validation as a part of the appraisal of Evidence conducted by a Verifier.
 This is in contrast to intuition that there has to be a unique Attester Identity per device.
 
-For DAA, the role of the Endorser is essentially the same, but they now provide Attester endorsement documents to the DAA Issuer rather than directly to the verifier. These Attester endorsement documents enable the Attester to obtain a credential from the DAA Issuer.
+For DAA, the role of the Endorser is essentially the same, but it now provides Attester endorsements to the DAA Issuer rather than directly to the Verifier. These Attester endorsements enable the Attester to obtain a credential from the DAA Issuer.
```

Comment:
"Attester endorsements" is a confusing term. Since the DAA Issuer contains a Verifier, the RATS Arch roles and conceptual messages remain consistent. The Attester Evidence is what convinces the DAA Issuer to issue a group credential. The group credential is new form of Endorsement that is created in response to an internal Attestation Result. I don't believe the draft needs to use the term "Attester endorsement" and lower case "endoorsement" seems like a misuse of the expected capitalization convention.